.github/workflows/ci.yml

```diff
@@ -3,24 +3,21 @@ on:
   pull_request:
 
 jobs:
-  DotNetFramework:
-    continue-on-error: true
-    runs-on: windows-latest
+  CI:
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Setup NuGet.exe
-        uses: nuget/setup-nuget@v1.0.5
-        with:
-          nuget-version: latest
-      - name: Setup MSBuild
-        uses: microsoft/setup-msbuild@v1.0.1
-      - name: Setup VSTest.console.exe
-        uses: darenm/Setup-VSTest@v1.1.1
-      - name: Restore Packages
-        run: nuget restore dropbox-sdk-dotnet\Dropbox.Api.sln
-      - name: Build
-        run: msbuild dropbox-sdk-dotnet\Dropbox.Api\Dropbox.Api.csproj -p:Configuration=Release
-      - name: Build Tests
-        run: msbuild dropbox-sdk-dotnet\Dropbox.Api.Unit.Tests\Dropbox.Api.Unit.Tests.csproj -p:Configuration=Debug
       - name: Run Unit Tests
-        run: vstest.console dropbox-sdk-dotnet\Dropbox.Api.Unit.Tests\bin\Debug\Dropbox.Api.Unit.Tests.dll
+        run: |
+          dotnet test \
```

Comment:
Does `dotnet` come preinstalled on all of the github actions hosts? Do we not need to install/setup `msbuild` or `vstest` or anything like that?

Reply:
Yep, looks like .NET Core 5 is available on all hosts.